### DISCUSSION TITLE
File List in alphabetical order

### DISCUSSION THREAD

**Discussion by amsharma9** [dail8859/NotepadNext#604] [Q&A]
I am on Ubuntu 22.04 platform. Is there some way to see "File List" visible on the left pane in alphabetical order? It's difficult to find and go to a particular file among the open files unless the list is sorted.

**Comment by dail8859** [comment]
On Windows I've always seen it sorted by file name. What does it appear it is sorted on for you?

**Reply by amsharma9** [reply]
Sorry for the late reply @dail8859. I haven't been watching my reply notification mails. On Ubuntu it isn't sorted. I have attached an image. HTH.
![NotepadNextFileList](https://github.com/user-attachments/assets/04da03cb-9ed7-45f2-93e1-99cd167712f3)

**Reply by dail8859** [reply]
Oh! Sorry. My mistake, I was thinking the "Folder as Workspace" dock. But you are referring to the File List.

That is (roughly) in the order that the files are opened in the main editor, e.g. left most editor is at the top, but nothing sorts them currently. I'll see what I can do about being able to sort that list.
